mandelbrot.rb

```diff
@@ -0,0 +1,47 @@
+require 'rubygems'
+require 'complex'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.